### PR TITLE
massdns 1.0.0 (new formula)

### LINKED ISSUE #91: Homebrew Formula
Is it possible to make `massdns` installable via [Homebrew](https://brew.sh/) for Mac OS?

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

fixes https://github.com/blechschmidt/massdns/issues/91

### CODE REVIEW

**Comment by chenrui333** [comment]
raised [the license issue,](https://github.com/blechschmidt/massdns/issues/94)

**Comment by carlocab** [comment]
Looks like they also have code that is licensed under GPL-2.0: https://github.com/blechschmidt/massdns/search?q=license

**Comment by chenrui333** [comment]
> Looks like they also have code that is licensed under GPL-2.0: https://github.com/blechschmidt/massdns/search?q=license

that code (`src/hashmap.h`) was adopted from android codebase under apache-2.0

File: Formula/massdns.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Massdns < Formula
+  desc "Particle library for 3D graphics"
```
Comment (blechschmidt) [inline Formula/massdns.rb:2]: The description seems to be a copy paste error.

Comment (carlocab) [reply Formula/massdns.rb:2]: Already fixed. ebc5b2f357be466fda209754b110e0daca8a49e4

**Pull request closed without merging**